flask_resources/content_negotiation.py

```diff
@@ -11,6 +11,7 @@
 
 from flask import request
 from werkzeug.datastructures import MIMEAccept
+from werkzeug.exceptions import NotAcceptable
 
 from .context import resource_requestctx
 from .errors import UnsupportedMimetypeError
```

Comment:
moderate: remove unused import?

Reply:
To be honest I might reintroduce it in the next commit. I was just scoping down. Will remove it for now though :+1:

Reply:
Yes!